cmd/verify/permissions/cmd.go

```diff
@@ -69,7 +69,12 @@ func run(cmd *cobra.Command, _ []string) {
 		if strings.Contains(fmt.Sprintf("%s", err), "STS") {
 			r.OCMClient.LogEvent("ROSAInitCredentialsSTS", nil)
 		}
-		r.Reporter.Errorf("Error creating AWS client: %v", err)
+		if strings.Contains(fmt.Sprintf("%s", err), "InvalidClientTokenId") {
```

Comment:
@willkutler What about code that is reliant on calling `Runtime.WithAWS()` i.e code that isn't building the AWS client directly? Are we handling the error consistently there as well?

Reply:
runtime.WithAWS calls Build() so the new placement of the error handler will resolve this as well